### PR TITLE
Github issue #154 | Jira EPM-745

### PR DESCRIPTION
**Pull request by microsarah**
change Associative Arrays to Objects

### CODE REVIEW

File: 11_chapter/08_lesson.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ If Floyd eats his lunch, does it affect anyone else's food? Nope! The *associati
 | Shannon   | soup          |
 | Josh      | pasta         |
 
-This is the basic principle underlying an **associative array** (also known in some languages as a *hash*). An associative array *associates* each value with a reference called a **key** (in this case the person's name). An important note is that, because each key/value pair is independent of any of the others, the array doesn't keep a consistent 'order' to its elements.
+This is the basic principle underlying an **myObject** (also known in some languages as a *hash*). An object *associates* each value with a reference called a **key** (in this case the person's name). An important note is that, because each key/value pair is independent of any of the others, the array doesn't keep a consistent 'order' to its elements.
```
Comment (jnappy) [inline 11_chapter/08_lesson.md]: @microsarah should this be **object** instead of **myObject** ?

**Comment by jnappy** [comment]
@microsarah I tagged you in one question.  Ready to merge, otherwise.

File: unreleased_chapters/06_chapter/02_lesson.md [hunk 2]
```diff
@@ -14,16 +14,16 @@ basicArray[2] = "Lake Oswego";
 console.log(basicArray.length);
 // --> Outputs 3, as expected
 
-var associativeArray = new Array();
-associativeArray['city'] = "Portland";
-associativeArray['state'] = "Oregon";
-associativeArray['country'] = "United States";
-console.log(associativeArray.length);
+var object = new Array();
```
Comment (jshawl) [inline unreleased_chapters/06_chapter/02_lesson.md]: `var object = {}`

File: unreleased_chapters/06_chapter/02_lesson.md [hunk 3]
```diff
@@ -14,16 +14,16 @@ basicArray[2] = "Lake Oswego";
...
+object['city'] = "Portland";
```
Comment (jshawl) [inline unreleased_chapters/06_chapter/02_lesson.md]: `object.city`

File: unreleased_chapters/06_chapter/02_lesson.md [hunk 4]
```diff
@@ -14,16 +14,16 @@ basicArray[2] = "Lake Oswego";
...
+object['state'] = "Oregon";
```
Comment (jshawl) [inline unreleased_chapters/06_chapter/02_lesson.md]: `object.state`

File: unreleased_chapters/06_chapter/02_lesson.md [hunk 5]
```diff
@@ -14,16 +14,16 @@ basicArray[2] = "Lake Oswego";
...
+object['country'] = "United States";
```
Comment (jshawl) [inline unreleased_chapters/06_chapter/02_lesson.md]: `object.country`

File: unreleased_chapters/06_chapter/02_lesson.md (hunk 2 above)
Comment (jnappy) [inline unreleased_chapters/06_chapter/02_lesson.md]: good catch @jshawl

**Pull request merged**